pkg/sparrow/controller_test.go

```diff
@@ -211,7 +213,7 @@ func TestChecksController_Reconcile(t *testing.T) {
 
 			cc.Reconcile(ctx, tt.newRuntimeConfig)
 
-			// iterate of the controllers checks and check if they are configured
+			// iterate of the controller's checks and check if they are configured
 			for _, c := range cc.checks.Iter() {
 				cfg := c.GetConfig()
```

Comment:
This method seems to only be used by tests. We may remove it from the interface in the future.